### ISSUE TITLE
Provider Button Shadows Clipped

### ISSUE THREAD

**Issue by Juchar** [firebase/FirebaseUI-Android#404]
### Environment Description

  * Android device: LG G5
  * Android OS version: 6.0.1
  * Google Play Services version: 9.8.77
  * Firebase/Play Services SDK version: _Na
  * FirebaseUI version: 1.0.0
  
#### Steps to reproduce:

  1. Start a new sign in flow:
```
startActivityForResult(
    AuthUI.getInstance()
        .createSignInIntentBuilder()
        .setProviders(Arrays.asList(new AuthUI.IdpConfig.Builder(AuthUI.EMAIL_PROVIDER).build(),
                                    new AuthUI.IdpConfig.Builder(AuthUI.GOOGLE_PROVIDER).build(),
                                    new AuthUI.IdpConfig.Builder(AuthUI.FACEBOOK_PROVIDER).build(),
                                    new AuthUI.IdpConfig.Builder(AuthUI.TWITTER_PROVIDER).build()))
        .build(),
    RC_SIGN_IN);
```
  
#### Observed Results:

Provider buttons are clipped (especially the shadows) on the left and right side:
![screenshot_2016-11-14-12-43-50](https://cloud.githubusercontent.com/assets/14292982/20263749/69ea7586-aa68-11e6-8c54-120be1395eb2.png)
  
#### Expected Results:

Not clipped buttons & shadows.
  
#### Possible fix:

Use the following attributes on the layout containing the buttons:
```
android:clipChildren="false"
android:clipToPadding="false"
```

**Comment by samtstern** [comment]
@Juchar thanks for the tip, would you be interested in submitting a PR with the fix?

**Comment by Juchar** [comment]
@samtstern I did actually not test this suggested fix, it's just an assumption as I once had a similar problem in one of my layouts, thats why I did not provide a PR.

**Comment by SUPERCILEX** [comment]
Yup, #407 fixes it:
![screenshot_20161114-163933](https://cloud.githubusercontent.com/assets/9490724/20288625/886aa7da-aa89-11e6-91c8-0396b7065eed.png)

**Comment by samtstern** [comment]
The fix for this issue has been released in version `1.0.1`.
